reports/_setup.qmd

```diff
@@ -73,6 +73,40 @@ if (metadata$run_id != params$run_id) {
   )
 }
 
+# Extract hash values from dvc.lock file
+dvc_lock <- read_yaml("../dvc.lock")
+dvc_lock_values <- c(
+  dvc_lock$stages$ingest$outs[[1]]$md5,
+  dvc_lock$stages$ingest$outs[[2]]$md5,
+  dvc_lock$stages$ingest$outs[[3]]$md5,
+  dvc_lock$stages$ingest$outs[[4]]$md5,
+  dvc_lock$stages$ingest$outs[[5]]$md5,
+  dvc_lock$stages$ingest$outs[[6]]$md5,
+  dvc_lock$stages$ingest$outs[[7]]$md5
+)
```

Comment:
**suggestion:** You can simplify this by loading only the `$outs` part of the list, then flattening the values, i.e. something like:

```r
dvc_lock_values <- unlist(read_yaml("../dvc.lock")$$stages$ingest$outs)
```

Reply:
This didn't quite get me there so I used a `sapply()`